### PR TITLE
Test diff new line edge case Hi Manny

### PR DESCRIPTION
**Pull request by MannyGrill**

### CODE REVIEW

File: newLine.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 test
-tesssssst
+tesssssst
```
Comment (MannyGrill) [inline newLine.txt:2]: k

File: noNewLine.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 test 1
 test 2 ooo
-test 3 eafef
\ No newline at end of file
+test 3 eafef
```
Comment (MannyGrill) [inline noNewLine.txt:3]: h

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # Pull Request Repository
-A repository used to test pull requests in GitKraken
+A repository used to test pull requests in GitKraken test
```
Comment (MannyGrill) [inline README.md:2]: ?
